main.nf

```diff
@@ -25,6 +25,7 @@ params.production = false
 params.schema_ignore_params = ""
```

Comment:
Nit: is there way to limit the boolean values to be boolean.  For example, I think chelsea has said that accidental mis-spellings of production or use_grs will trigger production pipeline.

Reply:
Could add checks, I'll look into it for nextflow

Reply:
Hmm i think this is avoidable now because we have drop down options for all of these parameters in Tower whereas before for the bpc pipeline launch that wasn't available.. You no longer need to type it out. 
<img width="854" alt="image" src="https://github.com/user-attachments/assets/d1de865c-5ece-46c1-bf19-7015533baea2" />